### PR TITLE
[Tests-only] check output of occ command

### PR DESCRIPTION
**Pull request by haribhandari07**
## Description

Required for PR https://github.com/owncloud/user_ldap/pull/497
## Related Issue

-  https://github.com/owncloud/core/issues/33293

## How Has This Been Tested?

- CI
- manual

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 1]
```diff
@@ -797,6 +798,34 @@ public function theAdministratorSetsLogRotateFileSizeUsingTheOccCommand($size) {
...
+	 *
+	 * @param PyStringNode $content
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theCommandOutputShouldBe(PyStringNode $content) {
+		$commandOutput = $this->featureContext->getStdOutOfOccCommand();
+		// removing blank lines
+		$commandOutput = \implode("\n", \array_filter(\explode("\n", $commandOutput)));
```
Comment (dpakach) [inline tests/acceptance/features/bootstrap/OccContext.php]: Is `array_filter` required here?

Comment (dpakach) [reply tests/acceptance/features/bootstrap/OccContext.php]: Ok looks like it filters anything that casts to `false` by default.

**Review by dpakach — COMMENTED** [review]
LGTM :+1:

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 2]
```diff
@@ -797,6 +798,34 @@ public function theAdministratorSetsLogRotateFileSizeUsingTheOccCommand($size) {
 		);
 	}
 
+	/**
+	 * @When the admin lists the enabled user backends using the occ command
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/OccContext.php]: move this step to user_ldap

**Review by phil-davis — APPROVED** [review]

**Pull request merged**